Draft of my consulting post. (#30)
jxnl/blog: docs/writing/posts/consulting.md

---
draft: False
date: 2024-01-22
slug: consulting
categories:
  - Personal
authors:
  - jxnl
---

# Indie Consulting

!!! note "Fragments"

    This is just a note to the reader that I like to write in a fragmented style using speech to text so if you see any errors or don't really understand what's going on feel free to just contribute and edit or leave a question on the github repo.

I specify indie consulting as something that is completely and wholly separate from the big-time consulting we hear about from those ridiculous institutions. Check out this video roasting [McKinsey](https://www.youtube.com/watch?v=AiOUojVd6xQ) From John Oliver to understand how I feel about many of these folks. Theres another great [video](https://www.youtube.com/watch?v=vZE0j_WCRvI).

<!-- more -->

> 

Reviewer: ```suggestion
## My Background
```

!!! warning "The failure of big time consulting"

    The difference between Indie Consulting and Big Time Consulting as I define it can be captured in one simple idea. Something that Steve Jobs once talked about:

    > "I don't think there's anything inherently evil in consulting. I think that without owning something over an extended period of time, like a few years, where one has a chance to take responsibility for one's recommendations, where one has to see one's recommendations through all action stages and accumulate scar tissue for the mistakes and pick oneself up off the ground and dust oneself off, one learns a fraction of what one can. Coming in and making recommendations and not owning the results, not owning the implementation, I think is a fraction of the value and a fraction of the opportunity to learn and get better."

> I strongly believe that if you're a young person you should not be consulted. You should join a big company or a small company in each shit. And the shit that you eat is the scar tissue that develops over time.

Reviewer: ```suggestion
I strongly believe that if you're a young person you should not be consulted. You should join a big company or a small company and each shit. And the shit that you eat is the scar tissue that develops over time.
```

!!! note "Confidence is the memory of success"

>     Most of my training did not come from obtaining a PhD. Instead, it primarily stems from my 2 years worth of experience at 7 internships, Academia like NYU Epidemiology and Public Health all the way up to Facebook, And companies that worked on marketing and sales and machine learning throughout the way, building up to my 5-year tenure at Stitch Fix.

Reviewer: ```suggestion
    Most of my training did not come from obtaining a PhD. Instead, it primarily stems from my 2 years worth of experience at 7 internships, from time spent in academia like NYU Epidemiology and Public Health all the way up to Facebook, and from companies that worked on marketing and sales and machine learning. All of this built up to my 5-year tenure at Stitch Fix.
```

---
draft: False
date: 2024-01-22
slug: consulting
categories:
  - Personal
authors:
  - jxnl
---

# What I Learned from Indie Consulting

!!! note "Fragments"

    This is just a note to the reader that I like to write in a fragmented style using speech to text so if you see any errors or don't really understand what's going on feel free to just contribute and edit or leave a question on the github repo.

I specify indie consulting as something that is completely and wholly separate from the big-time consulting we hear about from those ridiculous institutions. Check out this video roasting [McKinsey](https://www.youtube.com/watch?v=AiOUojVd6xQ) From John Oliver to understand how I feel about many of these folks. Theres another great [video](https://www.youtube.com/watch?v=vZE0j_WCRvI) that i saw on tiktok.

<!-- more -->

!!! warning "The failure of big time consulting"

>     The difference between Indie Consulting and Big Time Consulting as I define it can be captured in one simple idea. Something that Steve Jobs once talked about:

Reviewer 2: ```suggestion
    The difference between Indie Consulting and Big Time Consulting as I define it can be captured in one simple idea. Something that [Steve Jobs once talked about](https://www.youtube.com/watch?v=-c4CNB80SRc):
```

    > "I don't think there's anything inherently evil in consulting. I think that without owning something over an extended period of time, like a few years, where one has a chance to take responsibility for one's recommendations, where one has to see one's recommendations through all action stages and accumulate scar tissue for the mistakes and pick oneself up off the ground and dust oneself off, one learns a fraction of what one can. Coming in and making recommendations and not owning the results, not owning the implementation, I think is a fraction of the value and a fraction of the opportunity to learn and get better."

> I strongly believe that if you're a young person you should not be consulted. You should join a big company or a small company and just eat shit and develop the scar tissue that develops over time.

Reviewer 2: ```suggestion
I strongly believe that if you're a young person you should not be consulted. You should join a big company or a small company and just eat shit and develop the scar tissue that develops over time.

My friend and fellow indie consultant [Hamel Husain](https://twitter.com/HamelHusain) has another perspective:

> People without meaningful work experience in a particular industry don't add much value (and often negative value) as a consultant.  Large consulting companies like McKinsey, BCG, etc peddle a dangerous form of intellectual masturbation for dysfunctional management and organizations.  Ask yourself, what iconic companies have made impressive pivots or disrupted an industry **because of the advice of McKinsey**? 

>  On the other hand, _good_ consultants can offer specific services and add value in unique ways that employees cannot.  Let me give you some examples:
> - Procurement specialists who will re-negotiate all of your contracts and leverage the fact they are negotiating on behalf of many companies.  I knew someone who did this for large restaurant chains and was able to cut the price of everything from toilet paper to rent by 15% on average.  The suppliers wanted to play nice because this consultant had lots of clients.  Similarly, [Cory Quinn](https://twitter.com/quinnypig) will do this with your cloud bills.
> - In my [consulting practice around LLMs](https://parlance-labs.com/), I often provide vendors with their first customers, broker deals between open source maintainers and commercial platforms, introduce highly qualified candidates that get hired, and other strategic activities that involve working with many different organizations simultaneously.  These activities are in addition to specialized engineering work I do around LLMs.  I believe that the the engineering work alone, while valuable, would NOT justify me as a consultant in the long term.
```

I'll give an example if what I wrote [here](../../assessment.md). This is taken from Alan Weiss, and I've been able to implemented this with great success, heres some of his [examples](https://www.alanweiss.com/wp-content/uploads/Sample%20Proposals.pdf).

### Structure of the Proposal

- Situation: What is the current situation?
- Objectives: What are the desired outcomes?
- Values: Frame it in terms of the value equation.
- Metrics: How do we measure success?
- Timing
- Terms and conditions

Some callouts:

- Metrics: If it's a struggle to measure success, it'll be hard to identify value and charge for it. You're likely looking at milestones and implementation, which results in trading time for money, rather than execution for results.

- Situation: This is actually just hard, and you need to listen well and develop a skill in appraising the client's situation.

## Some stuff I've learned so far

### Time for money vs capital allocator

> The more high-level the client, the more I realized that they think in terms of capital allocation. They are not trading time for money; they are trading capital for time. And so, the exchange rate is different. Just like minimizing the time to success is important. If you can really capture the value of what you do, you'll likely always be charging too little. If you believe that you can actually drive a business outcome, for example, increase $100K ARR in 3 months, then you should be charging a percentage of that. It could be a deal if we see it as simply spending money to make money. Realy companies are throwing around so much capital in hopes of improving talent, their revenue, their product, their valuation.

Reviewer 3: I realized that the more high-level the client, the more they think in terms of capital allocation. They are not trading time for money; they are trading capital for discretionary time. And so, the exchange rate is weighted more toward time, and the same principle applies to the importance of minimizing the time to success.

### Niche Down

You should niche down sooner rather than later. In the beginning, I was just trying to help anybody. Startups, I considered banks, I did technical writing, dev rel, application development, and more. However, the more specific your positioning is, the sharper your message is, and the more likely you are to be able to capture value.

Lack of focus is fun, but context switching is expensive. I made 40k working with 4-5 clients, but I could easily achieve the same with fewer clients and charging more if they were more focused efforts.

I've now drilled down my niche to be around the thesis that "RAG applications are subpar recommendation systems, and I can bring years of experience to help you build a superior one." The second thesis I now have is that many teams have traditionally hired software engineers who are unfamiliar with building probabilistic software. Since I've also been doing that for many years, I can help them sidestep some of the pitfalls and mistakes that I've made.

> Now I focus on working with experienced founders who have raised a series A and are looking to build a RAG application and make at least 500k. They are prequalified based on 'hunger' and to benefit from what someone can bring to the value equation, rather than hiring a contractor as form of labor arbitrage.

Author: ```suggestion
Now, I focus on working with experienced founders who have experience raising a Series A and are looking to build a RAG application and reach an ARR of at least $500k. They are prequalified based on 'hunger' and are thus well-positioned to benefit from what someone can bring to the value equation, unlike less 'hungry' clients who may simply be looking to hire a contractor as a form of labor arbitrage.
```

!!! note "Stop offering to work for free"

    I know a lot of people in my DMs offering to work for me for free. But what they realize is that working for free is not good enough because they actually provide no value.

    1. Dream outcome: You have not shown me that you understand what my dream outcome is, and so you have no way of capturing what exactly it is I want.
    2. Perceived likelihood of success: You have not shown me that you have the skills to actually achieve this outcome.
    3. Perceived time to success: You have not shown me that you can do this in a timely manner. If anything, if you work for free, I have no faith that I can even control or manage your time.
    4. Perceived difficulty of success: More often than not, I need help because I do not have enough time. There's a good chance it might take you 8 hours to do something I could do in 1 hour, and instead now I'm stuck reviewing your work and trying to clarify something, wasting even more of my time.

### Personal anecdote from 10 years ago

> While, I formally just started an LLC in November of 2023. I would say my first few clients came in 2016. I had a bunch of friends who were Thiel fellows and starting their companies and they would often come to me to ask me for advice. At that point I'd already been very deep in a lot of machine learning and data science applications and spent thousands of hours reading papers over the years. I often worked for free or for dinner or every once in a while a couple hundred dollars on venmo.

Reviewer 4: > While I formally started an LLC...

> A number of friends starting companies in the [Thiel Fellowship program](https://thielfellowship.org/) would often ask me for advice.

> I didn't have much to offer except the knowledge I had built up, later I realised that I provided value on was **decreasing the perceived time to success**. I had already done thousands of hours of research. And if you would have spent 30 minutes to ask me a question, I could give you an answer that might take you a week to figure out. That was it. I was just saving them time.

Reviewer 4: > ...I had built up. Later, I realized that I provided value by **decreasing...

> ...thousands of hours of research. ~And~ If you would...

Lack of focus is fun, but context switching is expensive. I made 40k working with 4-5 clients, but I could easily achieve the same with fewer clients and charging more if they were more focused efforts.

I've now drilled down my niche to be around the thesis that "RAG applications are subpar recommendation systems, and I can bring years of experience to help you build a superior one." The second thesis I now have is that many teams have traditionally hired software engineers who are unfamiliar with building probabilistic software. Since I've also been doing that for many years, I can help them sidestep some of the pitfalls and mistakes that I've made.

> Now I focus on working with experienced founders who have raised a series A and are looking to build a RAG application and make at least 500k. They are prequalified based on 'hunger' and to benefit from what someone can bring to the value equation, rather than hiring a contractor as form of labor arbitrage.

Reviewer 3: Now, I focus on working with experienced founders who have experience raising a Series A and are looking to build a RAG application and reach an ARR of at least $500k. They are prequalified based on 'hunger' and are thus well-positioned to benefit from what someone can bring to the value equation, unlike less 'hungry' clients who may simply be looking to hire a contractor as a form of labor arbitrage.
